components/prism-ichigojam.js

```diff
@@ -0,0 +1,11 @@
+// according to the offical reference (EN)
+// https://ichigojam.net/IchigoJam-en.html
+Prism.languages.ichigojam = {
+	'string': /"(?:""|[!#$%&'()*,\/:;<=>?^_ +\-.A-Z\d])*"/i,
+	'comment': /\b(?:'|REM)(?:.*)?/i,
```

Comment:
I'm a bit confused here.
I kinda get that you want the have a word boundary _before_ the `REM`, but are you sure you want to remove the one that was _after_? This would make `REMABC` be highlighted as a comment, is it intended?

Also, `.*` is already a pattern that can match nothing, you don't need to make it optional with `?`.

Please double check this pattern, and see if something like `/(?:'|\bREM\b).*/i` wouldn't be more appropriate.

Reply:
Yes, based on its docs and real code, commands starting with ```REM```  is all comments, no space required.

And the regex ```(?:.*)?``` is my mistake, I'll correct it in the next commit.